### PR TITLE
EN: there is a continuous stream of misspellings

### PR DESCRIPTION
**Pull request by alama**
more words

### CODE REVIEW

File: Dialogue/Mid-Mission/ob_9900_0317.csv [hunk 1]
```diff
@@ -56,7 +56,7 @@ EnemyFront#2,"""They're coming from <br>the front! Watch out!"""
 EnemyRight#2,"""They're coming from <br>the right! Watch out!"""
 EnemyBack#2,"""They're coming from <br>behind! Watch out!"""
 EnemyLeft#2,"""They're coming from <br>the left! Watch out!"""
-Broken#2,"""Ahaha! Just when you fix it... it gets <br>wrecked again! You little S.O.Bs <br>are just loving this, aren't ya!?"""
+Broken#2,"""Ahaha! Just when you fix it... it gets <br>wrecked again! Those S.O.B <br>are just loving this, aren't ya!?"""
```
Comment (marrub--) [inline Dialogue/Mid-Mission/ob_9900_0317.csv]: This was correct before.

File: Misc/actor_name.csv [hunk 2]
```diff
@@ -1320,14 +1320,14 @@ Light03#1,"""Intermediate light technique.<br>Crystallize photons to form a sing
 Light04#1,"""Health restoration technique.<br>Heals wounds and restores lost health<br>with photon energy."""
 Light05#1,"""Status ailment cure technique.<br>Through the purifying effects of photons,<br>remove contaminants from the body."""
 Light06#1,"""Advanced light technique.<br>Create an extremely dense sphere of light<br>which automatically reacts to enemies."""
-Light07#1,"""Advanced light technique.<br>Crystallizes photons into droplets of light<br>which persue targets and explode."""
+Light07#1,"""Advanced light technique.<br>Crystallizes photons into droplets of light<br>which pursues targets and explode."""
```
Comment (marrub--) [inline Misc/actor_name.csv]: This should be "pursue"

File: Orders/Client/co01030106.csv [hunk 3]
```diff
@@ -1,2 +1,2 @@
 Title#0,"""Darker Occurrence Mystery"""
-Explanation#0,"""The Darker's numbers seem to be increasing.<br>I'm sure you've noticed this too. The<br>higher-ups are getting more nervous as<br>the reports come in. I've been asked to<br>survey the ruins.<br><br>They haven't told me any more than that<br>though. Why don't you come along?<br>Lunch is on me.<br><yellow>※S-rank clear with Zeno only.<c>"""
+Explanation#0,"""The number of Darker seem to be increasing.<br>I'm sure you've noticed this too. The<br>higher-ups are getting more nervous as<br>the reports come in. I've been asked to<br>survey the ruins.<br><br>They haven't told me any more than that<br>though. Why don't you come along?<br>Lunch is on me.<br><yellow>※S-rank clear with Zeno only.<c>"""
```
Comment (marrub--) [inline Orders/Client/co01030106.csv]: This should be "The Darkers' numbers"

File: Orders/Client/co01040615.csv [hunk 4]
```diff
@@ -1,2 +1,2 @@
 Title#0,"""Giant Evil Rare Creature"""
-Explanation#0,"""There has been a new sighting.<br>Alonagahda, a large Darker, has shown up<br>in the Ruins. We need someone who can<br>outwrestle the guy, and that would be<br>you.<br><br>I'm counting on you.<br><br><br><yellow>※Target Lv.41+ Alonagahda.<c>"""
+Explanation#0,"""There has been a new sighting.<br>Alonagahda, a large Darker, has shown up<br>in the Ruins. We need someone who can<br>out wrestle the guy, and that would be<br>you.<br><br>I'm counting on you.<br><br><br><yellow>※Target Lv.41+ Alonagahda.<c>"""
```
Comment (marrub--) [inline Orders/Client/co01040615.csv]: This should be "out-wrestle"

File: Story/Episode 4/ra_043100.csv [hunk 5]
```diff
@@ -3,7 +3,7 @@ ra_043100_0020#0,"""From the beginning, Hitsugi, while conducting our investigat
 ra_043100_0030#0,"""After all, there are things that can't be learned with just the analysis of intelligence being brought back."""
 ra_043100_0040#0,"""While learning about the threats of Phantoms from gathered intel, \""that\"" happened, during our preparation."""
 ra_043100_0050#0,"""Error Code: Olympia, was it? Forced both Hitsugi and Aru to return."""
-ra_043100_0060#0,"""Not only are they interferring with our intel gathering, there were detections of them trying to steal information from me in that brief moment."""
+ra_043100_0060#0,"""Not only are they interfering with our intel gathering, there were detection of them trying to steal information from me in that brief moment."""
```
Comment (marrub--) [inline Story/Episode 4/ra_043100.csv]: This should be reworded to "there were traces of them trying to steal information from me in that brief moment."

File: Story/Episode 4/st_046040.csv [hunk 6]
```diff
@@ -22,7 +22,7 @@ st_046040_0200#0,"""Don't push yourself, Kohri Washinomiya. If we lose you, Moth
 st_046040_0210#0,"""And... no one would be able to save your friend."""
 st_046040_0220#0,"""...Yes."""
 st_046040_0230#0,"""You're..."""
-st_046040_0240#0,"""I haven't met you conciously since Japan. Although, I've already allowed myself to examine the details of your body, very thoroughly."""
+st_046040_0240#0,"""I haven't met you ??? since Japan. Although, I've already allowed myself to examine the details of your body, very thoroughly."""
```
Comment (marrub--) [inline Story/Episode 4/st_046040.csv]: This should be "I haven't consciously met you since Japan."

File: Story/Episode 4/st_046050.csv [hunk 7]
```diff
@@ -37,8 +37,8 @@ st_046050_0350#0,"""Ophiel Herbert. Why did you kill Bethor, faking an order fro
 st_046050_0360#0,"""What do you see beyond the path you walk with Mother?"""
 st_046050_0370#0,"""Old man... What do you think about the current situation on Earth?"""
 st_046050_0380#0,"""To respond to a question with a question... You're acting more and more strange."""
-st_046050_0390#0,"""Humanity has declined. In terms of genetics, technology, mental capabilites... to the point where we're almost devolving."""
+st_046050_0390#0,"""Humanity has declined. In terms of genetics, technology, mental capabilities... to the point where we're almost devolving."""
 st_046050_0400#0,"""Then Ether and Mother appeared, old man."""
-st_046050_0410#0,"""A reshifting of humanity. For humanity's sake, I couldn't care less how dirty these hands become."""
+st_046050_0410#0,"""A making of humanity. For humanity's sake, I couldn't care less how dirty these hands become."""
```
Comment (marrub--) [inline Story/Episode 4/st_046050.csv]: This should be "a recreation of humanity" I think?

File: Tutorial/Message Pack/mp_000110.csv [hunk 8]
```diff
@@ -1,2 +0,0 @@
-name01#0,"""Message Pack"""
-text01#0,"""Recoilers turn into offensive weapons when attacked, rebounding away at high velocity."""
```
Comment (marrub--) [inline Tutorial/Message Pack/mp_000110.csv:2]: Why was this removed? I don't actually know why.

File: Tutorial/Window/wh_tl_np_010490.csv [hunk 9]
```diff
@@ -1,2 +1,2 @@
 name01#0,"""Daily Orders"""
-text01#0,"""This will explain about daily orders.<br><br>The daily orders you receive change everyday.<br><br>Unlike Client Orders that you receive from other individuals, these include orders that are reportable even while still working towards the objective. If that happens, you are able to get a reward appropriate to the amount you were able to complete at the time of the report."""
+text01#0,"""This will explain about daily orders.<br><br>The daily orders you receive change everyday.<br><br>Unlike Client Orders that you receive from other NPC's, these include orders that can be turned in while you're still working towards the objective."""
```
Comment (marrub--) [inline Tutorial/Window/wh_tl_np_010490.csv]: "NPC's" should be "NPCs"

File: UI/ui_craft.csv [hunk 10]
```diff
@@ -620,7 +620,7 @@ ExplainEnchantMenu#0,"""<c f0df60>You can create Data Drives and Timed Ability D
 ExplainRecipeSelect#0,"""<c f0df60>Please select a recipe to update your Data Drive.<c>"""
 ExplainEnchantStampConfirm#0,"""<c f0df60>This is the final Data Drive update confirmation window. You can affix various effects with support items.<c>"""
 ExplainStampSelect#0,"""<c f0df60>Please select a Data Drive to create Timed Ability Data.<c>"""
-ExplainEnchantStampItemConfirm#0,"""<c f0df60>This is the final Timed Ability Data creation window.  You can extend effect durations with support items.<c>"""
+ExplainEnchantStampItemConfirm#0,"""<c f0df60>This is the final Timed Ability Data creation window.  You can extend effect duration with support items.<c>"""
```
Comment (marrub--) [inline UI/ui_craft.csv]: This was correct before.

**Review by marrub-- — CHANGES_REQUESTED** [review]

File: Tutorial/Message Pack/mp_000110.csv (hunk 8 above)
Comment (alama) [reply Tutorial/Message Pack/mp_000110.csv:2]: There no "Recoiler", so I send the file back to the Files folder

File: Orders/Client/co01030106.csv [hunk 11]
```diff
@@ -1,2 +1,2 @@
 Title#0,"""Darker Occurrence Mystery"""
-Explanation#0,"""The number of Darker seem to be increasing.<br>I'm sure you've noticed this too. The<br>higher-ups are getting more nervous as<br>the reports come in. I've been asked to<br>survey the ruins.<br><br>They haven't told me any more than that<br>though. Why don't you come along?<br>Lunch is on me.<br><yellow>※S-rank clear with Zeno only.<c>"""
+Explanation#0,"""The Drakers' numbers seem to be increasing.<br>I'm sure you've noticed this too. The<br>higher-ups are getting more nervous as<br>the reports come in. I've been asked to<br>survey the ruins.<br><br>They haven't told me any more than that<br>though. Why don't you come along?<br>Lunch is on me.<br><yellow>※S-rank clear with Zeno only.<c>"""
```
Comment (marrub--) [inline Orders/Client/co01030106.csv]: Darkers, not Drakers D:<

File: Story/Episode 4/ra_043100.csv [hunk 12]
```diff
@@ -3,7 +3,7 @@ ra_043100_0020#0,"""From the beginning, Hitsugi, while conducting our investigat
 ra_043100_0030#0,"""After all, there are things that can't be learned with just the analysis of intelligence being brought back."""
 ra_043100_0040#0,"""While learning about the threats of Phantoms from gathered intel, \""that\"" happened, during our preparation."""
 ra_043100_0050#0,"""Error Code: Olympia, was it? Forced both Hitsugi and Aru to return."""
-ra_043100_0060#0,"""Not only are they interfering with our intel gathering, there were detection of them trying to steal information from me in that brief moment."""
+ra_043100_0060#0,"""There were traces of them trying to steal information from me in that brief monment"""
```
Comment (marrub--) [inline Story/Episode 4/ra_043100.csv]: Not the whole line, just that part of the sentence

File: Tutorial/Window/wh_tl_np_010490.csv [hunk 13]
```diff
@@ -1,2 +1,2 @@
 name01#0,"""Daily Orders"""
-text01#0,"""This will explain about daily orders.<br><br>The daily orders you receive change everyday.<br><br>Unlike Client Orders that you receive from other NPC's, these include orders that can be turned in while you're still working towards the objective."""
+text01#0,"""This will explain about daily orders.<br><br>The daily orders you receive change everyday.<br><br>Unlike Client Orders that you receive from other NPCs', these include orders that can be turned in while you're still working towards the objective."""
```
Comment (marrub--) [inline Tutorial/Window/wh_tl_np_010490.csv]: There shouldn't be an apostrophe there

**Review by marrub-- — CHANGES_REQUESTED** [review]

**Review by marrub-- — APPROVED** [review]
